### PR TITLE
refactor metadata to exports.data that is statically imported

### PR DESCRIPTION
**Pull request by jbolda**

### CODE REVIEW

File: pages/docs/some-react-code/index.jsx [hunk 1]
```diff
@@ -3,18 +3,18 @@ import Demo from './_Demo'
 import DocumentTitle from 'react-document-title'
 import { config } from 'config'
 
+exports.data = {
+  title: 'Some React Code',
+  date: `${new Date()}`
```
Comment (KyleAMathews) [inline pages/docs/some-react-code/index.jsx]: This won't actually work as we don't run the extracted code :-)

Comment (jbolda) [reply pages/docs/some-react-code/index.jsx]: Hmm, I assume you mean the date? That would make sense. I copied that from the OP, and only tried the title. :)

Should touch base with the OP why he needed that perhaps.

Comment (KyleAMathews) [reply pages/docs/some-react-code/index.jsx]: Yes — that's a confusing part of this — it's in a JS file but it's static. We could potentially eval the data file but that'd only partially help as it'd add additional sources of confusion e.g. referencing other variables wouldn't work.

Comment (KyleAMathews) [reply pages/docs/some-react-code/index.jsx]: Just change it to a static date (ISO 8601) and I'll commit this. In any case, it doesn't make sense to have a dynamic date in this context. Actually... just drop the date altogether. A documentation page wouldn't need it.

Comment (rattrayalex) [reply pages/docs/some-react-code/index.jsx]: (I'm the OP referred to above) 

I don't need the date, I was pseudo-copying from some other example. 

As mentioned in the aforementioned thread, this is the code I have running locally, which is not working.

Comment (jbolda) [reply pages/docs/some-react-code/index.jsx]: ( @rattrayalex the @ reply doesn't seem to auto fill in mobile and didn't want to butcher your name.)

I removed the date as it isn't needed. I did notice the version was pretty old, but that shouldn't matter with the ^. I did update it. That is the only other thing I can think to try. Everything that I submitted in the PR works on .28 with no issue that I can see.

@KyleAMathews did you want to keep those examples or no?

**Comment by KyleAMathews** [comment]
Just use the title. The other two fields will just confuse people as
they'll think that's a pattern to follow.

**Comment by jbolda** [comment]
Shoot, I meant to put that into two commits so I can roll the one back. I'll have to change it tonight or something.

**Comment by rattrayalex** [comment]
Thanks for doing this!!

**Comment by KyleAMathews** [comment]
Thanks!

**Pull request merged**